modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
 KV_MODE = JSON
 category = Mule Splunk Integration
 description = Mule Agent event information
-....
-
-[NOTE]
-If this file doesn't exist yet, you must create it.
-
-After making these changes, you must restart your Splunk instance for them to take effect.
+----
 
-.Configurable Fields
-|===
-|Field|Data Type|Description|Type|Default Value
+. Restart your Splunk instance for your changes to take effect.
 
-|`user`
-|String
-|Username to connect to Splunk.
-|Required
-|
 
-|`pass`
-|String
-|The password of the Splunk user.
-|Required
-|
+=== Configure Splunk Integration in Runtime Manager 
 
-|`host`
-|String
-|IP or hostname of the server where Splunk is running.
-|Required
-|
+To send Mule event notifications from the Runtime Manager agent to Splunk:
 
-|`port`
-|int
-|Splunk management port.
-|Optional
-|8089
+. If you plan to send data to Splunk using the HTTP Event Collector or over TCP, check the <<Splunk Prerequisites>>.
```

Comment:
```suggestion
. If you plan to send data to Splunk using the HTTP Event Collector or over TCP, verify that your environment meets <<Splunk prerequisites>>.
```

Reply:
This is incorrect - It's not environment setup. They have to actually perform steps on Splunk.